go-server/api/swagger.yaml

```
@@ -0,0 +1,90 @@
+openapi: 3.0.2
+info:
+  title: Web Server
+  description: A simple web server for managing users
+  contact:
+    email: dmanor@redhat.com
+  version: "1.0"
+servers:
+- url: http://127.0.0.1:3000
+paths:
+  /api:
+    post:
```

Maybe it's better to change the route to

```suggestion
  /api/users:
    post: # Attempt to register new user
      ...
    get:  # List all the registered users
      ...  
```

`/api` is too generic API endpoint route